flink-end-to-end-tests/run-nightly-tests.sh

```diff
@@ -257,7 +257,7 @@ function run_group_2 {
     LOG4J_PROPERTIES=${END_TO_END_DIR}/../tools/ci/log4j.properties
 
     MVN_LOGGING_OPTIONS="-Dlog.dir=${DEBUG_FILES_OUTPUT_DIR} -DlogBackupDir=${DEBUG_FILES_OUTPUT_DIR} -Dlog4j.configurationFile=file://$LOG4J_PROPERTIES"
-    MVN_COMMON_OPTIONS="-Dflink.forkCount=2 -Dflink.forkCountTestPackage=2 -Dfast -Pskip-webui-build"
+    MVN_COMMON_OPTIONS="-Dflink.forkCount=2 -Dflink.forkCountTestPackage=1 -Dflink.forkCountTablePlanner=1 -Dfast -Pskip-webui-build"
```

Comment:
What's the impact on the test time by executing the table tests serially?

Reply:
I think that difference is okay given the additional stability

Reply:
(are the numbers on the Alibaba-provided machines (e.g. here in `apache/flink`) in the same ballpark?)